src/olympia/amo/views.py

```diff
@@ -78,7 +79,10 @@ def handler403(request):
 
 @non_atomic_requests
 def handler404(request):
-    if request.path_info.startswith('/api/'):
+    if request.path_info.startswith('/api/v3/'):
```

Comment:
Yeah, I could do that instead. I do hope that if we ever bump the API version we'll grep for `v3` because it's already hardcoded in various places to handle CORS, prevent redirect to locale/app, etc :)